packages/terra-framework-docs/src/terra-dev-site/doc/worklist-data-grid/example/WorklistDataGridMaskedCell.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import WorklistDataGrid from 'terra-worklist-data-grid';
+import gridDataJSON from './gridMaskedCellData.json';
```

Comment:
I know we still need to update our existing examples with Worklist Style Data but in the spirit of documenting as we go, could you pare down the data object to what's necessary to show the feature and include that data in this file so it's viewable in the doc example.

Something as small as a header row and one or two body rows with some cells masked would be good.

Screenshot for reference. It doesn't need to be complex and rich like this but having a patient name
![image](https://github.com/cerner/terra-framework/assets/9022825/2d30f9e3-4fb3-49f7-92fe-6921220229c2)

Ex:
```javscript
const gridData = { 
  cols: [
    { id: "Column-0", displayName: "Patient"},
    { id: "Column-1", displayName: "Location" },
    { id: "Column-2", displayName: "Illness Severity" },
    { id: "Column-3", displayName: "Visit" },
    { id: "Column-4", displayName: "Allergy" },
    { id: "Column-5", displayName: "Primary Contact" },
    { id: "Column-6", displayName: "Generic Order Counts" },
    { id: "Column-7", displayName: "Patient Age" },
    { id: "Column-8", displayName: "Medication History" },
    { id: "Column-9", displayName: "My Relationship" },
    { id: "Column-10", displayName: "Forms" }
  ],
  rows: [
    {
      id: 1,
      cells: [
        { content: "Fleck, Arthur"},
        { content: "1007-MTN"},
        { content: "Unstable"},
        { content: "Inpatient, 2 months"},
        { content: ""},
        { content: "Quinzell, Harleen"},
        { content: ""},
        { content: "", isMasked: true},
        { content: "", isMasked: true},
        { content: "Admitting Physician"},
        { content: ""}
      ]
    },
    {
      id: 2,
      cells: [
        { content: "Wayne, Bruce"},
        { content: "1007-MTN-DR"},
        { content: "Stable"},
        { content: "Outpatient, 2 days"},
        { content: "Phytochemicals"},
        { content: "Grayson, Richard"},
        { content: ""},
        { content: ""},
        { content: "", isMasked: true},
        { content: "Admitting Physician"},
        { content: ""}
      ]
    }
  ]
}
```

Reply:
I made the requested updates for the dev site.